### PR TITLE
s_reception_screen_measuring_device: fix device release

### PR DESCRIPTION
**Pull request by mmequignon**
related to https://github.com/OCA/wms/pull/214

### CODE REVIEW

**Review by simahawk — APPROVED** [review]

File: stock_reception_screen_measuring_device/models/stock_reception_screen.py [hunk 1]
```diff
@@ -18,6 +18,11 @@ class StockReceptionScreen(models.Model):
         store=True,
         help="Indicates if the package have any measurement missing.",
     )
+    display_package_dimensions = fields.Char(
```
Comment (sebalix) [inline stock_reception_screen_measuring_device/models/stock_reception_screen.py:21]:
```suggestion
    package_dimensions = fields.Char(
```
No need to add a verb here.

File: stock_reception_screen_measuring_device/models/stock_reception_screen.py [hunk 2]
```diff
@@ -33,6 +38,22 @@ def _compute_scan_requested(self):
...
+        "product_packaging_id.width",
+        "product_packaging_id.height",
+    )
+    def _compute_package_dimensions(self):
+        for record in self:
+            pack = record.product_packaging_id
+            dimension_values = [pack.lngth, pack.height, pack.width]
+            if all(dimension_values):
+                record.display_package_dimensions = " x ".join(
+                    [str(val) for val in dimension_values]
```
Comment (sebalix) [inline stock_reception_screen_measuring_device/models/stock_reception_screen.py]: Add mm unit would be nice
```suggestion
                    [f"{str(val)}mm" for val in dimension_values]
```

File: stock_reception_screen_measuring_device/models/stock_reception_screen.py [hunk 3]
```diff
@@ -18,6 +18,11 @@ class StockReceptionScreen(models.Model):
         store=True,
         help="Indicates if the package have any measurement missing.",
     )
+    display_package_dimensions = fields.Char(
+        string="Dimensions (lxhxw)",
+        compute="_compute_package_dimensions",
+        help="Dimensions of the package (length x height x width)",
```
Comment (sebalix) [inline stock_reception_screen_measuring_device/models/stock_reception_screen.py]:
```suggestion
        help="Dimensions of the package in mm (length x height x width)",
```

File: stock_reception_screen_measuring_device/models/stock_reception_screen.py (hunk 1 above)
Comment (mmequignon) [reply stock_reception_screen_measuring_device/models/stock_reception_screen.py:21]: "display" in "display_name" is not a verb.
Here it's just to say that's dimensions meant to be displayed.
But I can remove it if you still want it to be removed.

**Comment by simahawk** [comment]
@mmequignon pls address @sebalix comments but the "display" one ;)

File: stock_reception_screen_measuring_device/models/stock_reception_screen.py (hunk 1 above)
Comment (sebalix) [reply stock_reception_screen_measuring_device/models/stock_reception_screen.py:21]: If it "means to be displayed" then it is a boolean (to display or not something), but if the field contains qualitative information regarding the current record (we are stocking a string here), then `package_dimensions` is enough, no need to add more.
It's my POV, non blocking.

**Review by sebalix — APPROVED** [review]

**Pull request merged**